Update LaMetric documentation for refactor (#23800)
home-assistant/home-assistant.github.io: source/_integrations/lametric.markdown

---
title: LaMetric
description: Instructions on how to integrate LaMetric TIME with Home Assistant.
ha_category:
  - Notifications

> ha_iot_class: Local Polling

Reviewer: The core integration manifest says local_push. Should that change too?

Author: Yeah it will change back in a next PR... currently it does only notifications, so it is push. But the next PR introduces the coordinator, device and entities.

But at this point, you are right 👍 

```suggestion
ha_iot_class: Local Push
```

  trigger:
    platform: state
    entity_id: device_tracker.tom_mobile
    from: "not_home"
    to: "school"
  action:
    service: notify.my_lametric
    data:
      message: "Tom has arrived at school!"
      data:
        sound: "notification"
        icon: "51"
        cycles: 0
        priority: "critical"
        icon_type: "info"
```

## List of notification sounds

The following notification sounds can be used with the `sound` parameter on
notify service calls:

- `alarm1`
- `alarm10`
- `alarm11`
- `alarm12`
- `alarm13`
- `alarm2`
- `alarm3`
- `alarm4`
- `alarm5`
- `alarm6`
- `alarm7`
- `alarm8`
- `alarm9`
- `bicycle`
- `car`
- `cash`
- `cat`
- `dog`
- `dog2`
- `energy`
- `knock-knock`
- `letter_email`
- `lose1`
- `lose2`
- `negative1`
- `negative2`
- `negative3`
- `negative4`
- `negative5`
- `notification`
- `notification2`
- `notification3`
- `notification4`
- `open_door`
- `positive1`
- `positive2`
- `positive3`
- `positive4`
- `positive5`
- `positive6`
- `statistic`
- `thunder`
- `water1`
- `water2`
- `win`
- `win2`
- `wind_short`
- `wind`

## Manual automatic import configuration

If you prefer not to use the Home Assistant account linking service, you

> can do so manually.

Reviewer: ```suggestion
can set up the LaMetric application manually.
```

> However, please note! At this point, is it easier to choose the "Enter manually"

Reviewer: ```suggestion
However, please note! At this point, it is easier to choose the "Enter manually"
```

option during the integration setup; this also avoids the use of the account
linking service and doesn't need all the steps below either.

> If you still want to set up your own account linking service for importing

Reviewer: ```suggestion
If you still want to set up your own LaMetric application for importing
```
